tests/Integration/apis/compute_small/v1/compute_small.proto

```diff
@@ -0,0 +1,692 @@
+// Copyright 2020 Google LLC
```

Comment:
Looks like overall 14 new (mostly big) files were added to test the single max_results/page_size difference as a feature. This is a dangerous path (if we add so much stuff for every simple feature the tests will very quickly blow up - we ended up having exactly this problem for gapic-generator monolith). 

Can we just reuse the existing goldens for this? No need to introduce the whole compute_small thing in gapic-generator-php test infra. It is subjective, but maybe just not having the integration test for `max_results` is just ok  and would lead to a more maintainable code (as long as it is tested at least by unit tests).

Reply:
I've removed the `asset_rest` in favor of this, which will be the sole REGAPIC integration test (as I came across the `compute_small` protos only after writing #277.)